shared/src/commonMain/kotlin/org/hyperskill/app/logging/sentry/SentryMessageStringFormatter.kt

```diff
@@ -0,0 +1,10 @@
+package org.hyperskill.app.logging.sentry
+
+import co.touchlab.kermit.Message
+import co.touchlab.kermit.MessageStringFormatter
+import co.touchlab.kermit.Severity
+import co.touchlab.kermit.Tag
+
+internal interface SentryMessageStringFormatter : MessageStringFormatter {
```

Comment:
Do we need a dedicated interface for a message formatter? In my opinion, it is redundant.

Reply:
We need this because `MessageStringFormatter` provides default formatting implementation that we are using:
```kotlin
interface MessageStringFormatter {
    fun formatSeverity(severity: Severity) = "$severity:"
    fun formatTag(tag: Tag) = "(${tag.tag})"
    fun formatMessage(severity: Severity?, tag: Tag?, message: Message): String {
        // Optimize for Android
        if(severity == null && tag == null)
            return message.message

        val sb = StringBuilder()
        if (severity != null) sb.append(formatSeverity(severity)).append(" ")
        if (tag != null && tag.tag.isNotEmpty()) sb.append(formatTag(tag)).append(" ")
        sb.append(message.message)

        return sb.toString()
    }
}
```